sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/diagnostics/CosmosItemIdentifier.scala

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.spark.diagnostics
+
+import com.azure.cosmos.models.PartitionKey
+
+case class CosmosItemIdentifier(itemId: String, partitionKeyValue: PartitionKey)
```

Comment:
I think we should add (at least otionally) eTag here?

Reply:
sure. thanks for the comment. I am tracking this in an upcoming PR.